cpp/example_code/s3/delete_objects.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+/**
+ * Before running this C++ code example, set up your development environment, including your credentials.
+ *
+ * For more information, see the following documentation topic:
+ * https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started.html
+ *
+ * For information on the structure of the code examples and how to build and run the examples, see
+ * https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started-code-examples.html.
+ *
+ * Purpose
+ *
+ * Demonstrates using the AWS SDK for C++ to delete multiple objects in a Simple Storage
+ * Service (Amazon S3) bucket.
+ *
+ */
+
+#include <iostream>
+#include <aws/core/Aws.h>
+#include <aws/s3/S3Client.h>
+#include <aws/s3/model/DeleteObjectsRequest.h>
+#include "awsdoc/s3/s3_examples.h"
+
+//! Routine which demonstrates deleting multiple objects in an Amazon S3 bucket.
+/*!
+  \sa DeleteObjects()
+  \param objectKeys: Vector of object keys.
+  \param fromBucket: Name of a bucket with an object to delete.
+  \param clientConfig: Aws client configuration.
+*/
+
```

Comment:
I believe that Aws should be AWS here, because in this case, it's a text description, not an argument or API name.